qcodes/tests/drivers/test_keysight_34465a.py

```diff
@@ -11,6 +13,28 @@ def driver():
     keysight_sim = Keysight_34465A('keysight_34465A_sim',
                                    address='GPIB::1::INSTR',
                                    visalib=visalib)
+
+    try:
+        yield keysight_sim
+    finally:
+        keysight_sim.close()
+
+
+@pytest.fixture(scope='function')
+def driver_volt(val_volt):
+    keysight_sim = Keysight_34465A('keysight_34465A_sim',
+                                   address='GPIB::1::INSTR',
+                                   visalib=visalib)
```

Comment:
keysight_sim is already defined, this should not be here I guess. Probably will fix the problem. @lakhotiaharshit

Reply:
Did not helped. But thanks to your suggestion, I removed some code duplication.